doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,72 @@
+.. rn:: 5.6.26-74.0
+
+==============================
+ |Percona Server| 5.6.26-74.0 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.26-74.0 on September 15th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.26-74.0/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.26 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-26.html>`_, including all the bug fixes in it, |Percona Server| 5.6.26-74.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.26-74.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.26-74.0>`_.
+
+New Features
+============
+
+ |TokuDB| storage engine `source <https://github.com/percona/tokudb-engine.git>`_ has been merged into the |Percona Server| code. |TokuDB| storage engine version has been replaced by the |Percona Server| version. 
+ 
+ |TokuDB| Hot Backup has been renamed to |Percona TokuBackup| and it is now `open source <https://github.com/percona/Percona-TokuBackup.git>`_. Source `code <https://github.com/percona/tokudb-backup-plugin>`_ has been integrated into |Percona Server| code as git submodule. 
+
+ *Tokutek Fractal Tree* has been renamed to |Percona FT| and its `source code <https://github.com/percona/PerconaFT.git>`_ has been integrated into |Percona Server| code as git submodule. 
+
+ Google `SNAPPY <http://google.github.io/snappy/>`_ compression/decompression algorithm is now available as :ref:`tokudb_compression` table format.
+
+ |Percona Server| now supports changing the :variable:`server_id` variable :ref:`per session <per_session_server-id>`, by implementing the new :variable:`pseudo_server_id` variable. This feature is also fixing upstream bug :mysqlbug:`35125`.
+
+ |Percona Server| has temporarily disabled savepoints in triggers and stored functions. The reason is that even having fixed bug :bug:`1438990` and bug :bug:`1464468` we have found more cases where savepoints in triggers break binary logging and replication, resulting in server crashes and broken slaves. This feature will be disabled until the above issues are properly resolved.
+
+ ``LOCK BINLOG FOR BACKUP`` can be modified to flush (and redo-log) the current binary log coordinates to |InnoDB|. 
```

Comment:
LTFB now flushes the current binary log coordinates to InnoDB. Thus, under active LTFB, the binary log coordinates in InnoDB are consistent with its redo log and any non-transactional updates (as the latter are blocked by LBFB). It is planned that this change will enable XtraBackup to avoid issuing the more invasive LBFB command under some circumstances.